OpenOversight/app/model_imports.py

```diff
@@ -0,0 +1,307 @@
+from OpenOversight.app.models import (
+    Assignment,
+    Incident,
+    LicensePlate,
+    Link,
+    Location,
+    Officer,
+    Salary,
+    db,
+)
+from OpenOversight.app.main import choices
+import dateutil
+from OpenOversight.app.utils import get_or_create, str_is_true
+from OpenOversight.app.validators import state_validator, url_validator
+from typing import Optional, Sequence, Tuple, Union, Dict, Any, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import datetime  # noqa
+
+
+def validate_choice(
+    value: Optional[str], given_choices: Sequence[Tuple[str, str]]
+) -> Optional[str]:
+    if value:
+        for label, choice_value in given_choices:
+            if value.lower() in [choice_value.lower(), label.lower()]:
+                return label
+        print("'{}' no valid choice of {}".format(value, str(given_choices)))
+    return None
+
+
+def parse_date(date_str: Optional[str]) -> Optional["datetime.date"]:
+    if date_str:
+        return dateutil.parser.parse(date_str).date()
```

Comment:
good stuff, I'm a fan of this lib :+1:

Reply:
nit: why `"datetime.date"` above? `Optional[datetime.date]` also works, and is what I'm used to; didn't know you could do the quoted version

Reply:
Without quotes I need to actually import it, this way I can put the import under `if TYPE_CHECKING`, so it's not imported at runtime. Not really relevant with small built-in libraries like this one, but I like to do it until we have at least python 3.7 and `from __future__ import annotations` which circumvents the problem: https://www.python.org/dev/peps/pep-0563/